Documentation/Development/Index.rst

```diff
@@ -23,3 +23,56 @@ Hooks
 	Hooks/ExtraDataProcessing
 	Hooks/ExtraMergeField
 	Hooks/GetGroupsProcessing
+
+Continuous Deployment
+-----
+To support continuous deployment, it is possible to set a configuration via PHP through the following code in Additional_Configuration.php or ext_localconf.php of your extension.
+
+.. toctree::
```

Comment:
why `.. toctree::` ?

Reply:
This is `AdditionalConfiguration.php`. Please use proper reST markup for referencing files. In addition, you use the ConfigurationManager to set the configuration of the extension, but that's not how one is supposed to write custom configuration in `AdditionalConfiguration.php`. Why not directly assigning the value to the global array as it is usually done?